models/staging/ga4/stg_ga4__items.sql

```diff
@@ -0,0 +1,41 @@
+{{
+    config(
+        enabled = false,
+    )
+}}
+
+ with items_with_params as (
+    select
+        event_key,
```

Comment:
I think it would be useful to keep the event_name in the select clause here. What do you think? If someone goes here looking for purchased items, they'll want to set a where clause on `event_name = 'purchase'`